docs/topics/api/overview.rst

```diff
@@ -223,3 +223,4 @@ v4 API changelog
   https://github.com/mozilla/addons-server/issues/8794
 * 2018-07-18: added ``previews`` property to discovery API ``addons`` object. This change was also backported to the `v3` API. https://github.com/mozilla/addons-server/issues/8863
 * 2018-07-20: dropped ``downloads`` property from the collection add-ons results. https://github.com/mozilla/addons-server/issues/8944
+* 2018-08-09: added ``is_developer_reply`` property to ratings. This changed was also backported to the `v3` API. https://github.com/mozilla/addons-server/issues/8993
```

Comment:
Will update as it won't make the tag.

Reply:
The sooner we get #8899 done the sooner we can stop this risky modification of the "frozen" v3 API 😿